src/engraving/rw/write/twrite.cpp

```diff
@@ -429,7 +429,7 @@ void TWrite::writeStyledProperties(const EngravingItem* item, XmlWriter& xml)
 
 void TWrite::writeItemProperties(const EngravingItem* item, XmlWriter& xml, WriteContext& ctx)
 {
-    if (!MScore::testMode && !item->score()->isPaletteScore()) {
+    if (!MScore::testMode && !item->score()->isPaletteScore() && !ctx.clipboardmode()) {
```

Comment:
Thank you, I think this is a good thing to do. Does this mean that files saved with versions up to current master may contain different items with the same ID? Cause that's a small problem (and a thing that's good to know, in any case).

Reply:
Yes - anything copied & pasted with range select will have a duplicate id unfortunately.